### DISCUSSION TITLE
Position of the table of content

### DISCUSSION THREAD

**Discussion by Moldex77** [Zavy86/WikiDocs#183] [Ideas]
Hey guys why not use the table of content in static mode on the right of the website in order to have the doc guideline permanently.
This will permitt to navigate efficiently for long articles. Many thanks for the accomplish work

![image](https://github.com/user-attachments/assets/f2200e00-faec-4a70-aa83-881a60751a1f)

**Comment by Zavy86** [comment]
it's a good idea, I think we can do it with css..
